prometheus-exporters/esxi-exporter/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: esxi-exporter
+  namespace: infra-monitoring
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: {{ .Values.listen_port | quote }}
+    prometheus.io/targets: {{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus | quote }}
+spec:
+  selector:
+    app: {{ include "infraMonitoring.name" . }}
```

Comment:
Looks like this could not be resolved?
https://ci.eu-de-2.cloud.sap/builds/98120

Reply:
jup. resolved.